src/shared/default-routes.js

```diff
@@ -40,16 +41,16 @@ const DefaultRoutes = ({
       }
       <Route
         path='/'
-        component={FrontPage || Page}  />
+        component={fetchData(FrontPage || Page || Error)}  />
       <Route
         path=':page(/:subpage)'
-        component={Page} />
+        component={fetchData(Page || Error)} />
```

Comment:
Right, but wouldn't that mean if they passed an `Error` component they'll never see the `MissingView`? Just curious.